astroid/protocols.py

```diff
@@ -167,6 +167,9 @@ def _multiply_seq_by_int(
     context: InferenceContext,
 ) -> _TupleListNodeT:
     node = self.__class__(parent=opnode)
+    if isinstance(other.value, int) and other.value > 1e8:
+        node.elts = [nodes.Const(NotImplemented)]
```

Comment:
Shouldn't this be Uninferable instead of NotImplemented ?

Reply:
I suppose that's better than implying there's one element in the list.